driver-mapping/src/main/java/com/datastax/driver/mapping/MethodMapper.java

```diff
@@ -92,6 +88,30 @@ public void prepare(MappingManager manager, PreparedStatement ps) {
         }
     }
 
+    // Checks the method parameters against the query's bind variables
+    private void validateParameters() {
+        if (method.isVarArgs())
+            throw new IllegalArgumentException(String.format("Invalid varargs method %s in @Accessor interface", method.getName()));
+
+        ColumnDefinitions variables = statement.getVariables();
+        Set<String> names = Sets.newHashSet();
```

Comment:
We already have a private field `byName`in `ColumnDefintions`, why not simply expose it?